parcels/kernels/EOSseawaterproperties.py

```diff
@@ -0,0 +1,348 @@
+"""Collection of pre-built eos sea water property kernels"""
+import math
+
+__all__ = ['Pressure_from_LatDepth', 'Adtg', 'Ptemp_from_Temp', 'Temp_from_Ptemp']
+
+
+def Pressure_from_LatDepth(particle, fieldset, time):
+    """
+    Calculates pressure in dbars from depth in meters and latitude.
+
+    Returns
+    -------
+    p : array_like
+           pressure [db]
+
+    References
+    ----------
+    .. [1] Saunders, Peter M., 1981: Practical Conversion of Pressure to Depth.
+       J. Phys. Oceanogr., 11, 573-574.
+       doi: 10.1175/1520-0485(1981)011<0573:PCOPTD>2.0.CO;2
+
+    """
+    # Angle conversions.
+    deg2rad = math.pi / 180.0
+
+    X = math.sin(max(particle.lat * deg2rad, -1*particle.lat * deg2rad))
+    C1 = 5.92e-3 + math.pow(X, 2) * 5.25e-3
+    particle.pressure = ((1 - C1) - math.pow(((math.pow((1 - C1), 2)) - (8.84e-6 * particle.depth)), 0.5)) / 4.42e-6
+
+
+def Adtg(particle, fieldset, time):
```

Comment:
I know this is short and 'dt g' is temperature gradient and so - now. But could you please give this Kernel a longer, more concise name ? the name is a bit cryptic.

Reply:
The name is changed to: Adiabtic_temperature_gradient(..). Is that alright?

Reply:
yeah, that's good - distinct. Thanks.